server/controller/processors/goalProcessor.ts

```diff
@@ -64,14 +70,21 @@ async function putGoalFeedback(req: Request, res: Response) {
         return;
     }
     res.sendStatus(StatusCode.OK);
+    const goal = await goalAPI.getGoalById(Number(req.params.id))
+    const goals = JSON.stringify(goal);
```

Comment:
There shouldn't be more than one goal that results from parsing the goal with the Id. Perhaps a better name would be "goalString" or something. I dunno, it's incredibly minor, but I just thought this name was a little misleading.